crypter_test.go

```diff
@@ -694,22 +696,6 @@ func generateTestKeys(keyAlg KeyAlgorithm, encAlg ContentEncryption) []testKey {
 	panic("Must update test case")
 }
 
-func RunRoundtripsJWE(b *testing.B, alg KeyAlgorithm, enc ContentEncryption, zip CompressionAlgorithm, priv, pub interface{}) {
```

Comment:
Huh, this was unused? Must've forgotten to clean this up during a refactor I guess.